openeo/capabilities.py

```diff
@@ -1,10 +1,92 @@
+import contextlib
 from abc import ABC
-from distutils.version import LooseVersion
+import re
 from typing import Union
 
 # Is this base class (still) useful?
 
 
+class LooseVersion:
+    """A version number consists of a series of numbers,
```

Comment:
If I understand correctly this is a straight copy from the original `distutils.version.LooseVersion`.
I think that should be noted in the docstring.

Moreover it now looks unnecessary and overkill to have both LooseVersion and ComparableVersion defined here. I think it makes more sense to merge both classes to a single class

Reply:
Makes sense. Fixed.